rebar.config

```diff
@@ -10,7 +10,7 @@
 
 {deps, [
         {blockchain, {git, "https://github.com/helium/blockchain-core.git",
-                      {branch, "master"}}},
+                      {branch, "pevm/testnet-integration1"}}},
```

Comment:
reminder, switch to master when branch lands. also looks like no corresponding PR on core side?

Reply:
most of the core stuff has already landed, and I think that we have landed all the subsequent relevant PRs as well.  We do need to upgrade, though.